src/main.rs

```diff
@@ -33,10 +36,25 @@ async fn main() {
         .get_info(tonic_lnd::lnrpc::GetInfoRequest {})
         .await
         .expect("failed to get info");
+    let info_clone = info.into_inner().clone();
+    let info_clone_2 = info_clone.clone();
```

Comment:
I don't think we need either of these clones because we only borrow from `info`?

Reply:
So if we remove the clone, I can't access info.features on line 45, because info is moved on line 42 (because the use of into_inner() moves it, it seems). 

But to your point, I just cleaned this up and got rid of one of the clones since looks like that was unnecessary!